indigo/distribution.yaml

```diff
@@ -12684,6 +12684,21 @@ repositories:
       url: https://github.com/turtlebot/turtlebot_create_desktop.git
       version: indigo
     status: maintained
+  turtlebot_exploration_3d:
+    doc:
+      type: git
+      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
+      version: indigo
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
+      version: 0.0.1
```

Comment:
Sorry I am confused, thought I need to pull request first and then
bloom-release.
The documentation of "get involved" is a bit confusing...

2016-08-25 17:22 GMT-04:00 William Woodall <notifications@github.com>:

> In indigo/distribution.yaml
> <https://github.com/ros/rosdistro/pull/12504#discussion_r76327275>:
>
> > @@ -12684,6 +12684,21 @@ repositories:
> >        url: https://github.com/turtlebot/turtlebot_create_desktop.git
> >        version: indigo
> >      status: maintained
> > +  turtlebot_exploration_3d:
> > +    doc:
> > +      type: git
> > +      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
> > +      version: indigo
> > +    release:
> > +      tags:
> > +        release: release/indigo/{package}/{version}
> > +      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
> > +      version: 0.0.1
>
> Did you run bloom-release for Indigo? If not the release section will not
> work. You can add the doc and source sections without releasing it with
> bloom.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ros/rosdistro/pull/12504/files/a501576c1230317cfa52ace42d339d1a9ba47ab5#r76327275>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKFBr82faVG11o4IOk8XCYw4hvDWan8Lks5qjgeYgaJpZM4Jtg83>
> .
>